apps/frontend/src/components/Decorators/Icon.tsx

```diff
@@ -1,22 +1,25 @@
-import { kebabToPascal } from '@react-site-editor/functions';
 import { lazy, Suspense } from 'react';
+import { kebabToPascal } from '@react-site-editor/functions';
 import type { IconName } from '@/libs/types/icons.type';
+import type { TooltipPlace } from '@/libs/types/tooltip.type';
 
 interface IconProps {
     name: IconName;
     description: string;
     className?: string;
+    descriptionPlace?: TooltipPlace;
     onClick?: () => void;
 }
 
 const Icon: React.FunctionComponent<IconProps> = (props) => {
     const DynamicIcon = lazy(
-        () => import(`../Icons/${kebabToPascal(props.name)}`)
+        () => import(`../Icons/${kebabToPascal(props.name)}.tsx`)
```

Comment:
This change is to remove the warning from `Vite`.
See the [limitations](https://github.com/rollup/plugins/tree/master/packages/dynamic-import-vars#limitations) of imports.